source/crud/crud.rst

```diff
@@ -54,10 +54,10 @@ Iterable
 Guidance
 --------
 
-Documentation
+documention
```

Comment:
I think you search-and-replaced too much when correcting the original documentation/document typo. Some of these instances should be "documentation".